src/modules/reporting/components/reporting-report/reporting-report.jsx

```diff
@@ -123,6 +123,7 @@ export default class ReportingReport extends Component {
             location={p.location}
             history={p.history}
             cardStyle="single-card"
+            hideReportEndingIndicator
```

Comment:
I can't find this being set anywhere. Where should it be set?

Reply:
It's not set anywhere. Passing it in like this is treated as `hideReportEndingIndictor={true}`. I thought it was common practice for us to do it this way? If not, I can be more verbose.

Reply:
Really? I did not know that! Cool trick! I figured it came through as undefined.
If that is the case I'm totally happy with it.